pkg/matchers.go

```diff
@@ -18,15 +18,21 @@ func IsInstalled(pkgs ...Pkg) error {
 			return fmt.Errorf("Unable to look up %s: no package found", p)
 		}
 
-		if p.Version() == "" {
+		if p.Version() == nil {
 			continue
 		}
 
+	CheckPkgs:
 		for _, foundPkg := range foundPkgs {
-			if foundPkg.Version() == p.Version() {
-				continue
+			for _, foundPkgVersion := range foundPkg.Version() {
+				for _, pkgVersion := range p.Version() {
```

Comment:
This is a little bit missleading. `p.Version()` will return several versions? Is it possible that i want to check for more than 1 version installed?

I would say that this should be let for implementation to the client. I mean, if you want to check that more than 1 package version is installed you do the loop, not us.

Or... change the name to `p.Versions()`? Perhaps something like: `FirstVersion(), LastVersion() & Versions()`?